core/ledger/package.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "core-ledger",
```

```suggestion
    "name": "core-ledger-lib",
```

Implies that the folder name changes accordingly